.github/workflows/install-ssf.yaml

```
@@ -0,0 +1,18 @@
+name: CI
+on:
+  - pull_request
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: build example and deploy to minikube
+    steps:
+    - uses: actions/checkout@v2
+    - name: Start minikube
+      uses: medyagh/setup-minikube@master
+    - name: Try the cluster !
+      run: kubectl get pods -A
+    - name: Deploy SSF to minikube
+      run:
+        ./platform/10-tekton-setup.sh
+        ./platform/11-tekton-chains.sh
```

Since we have a `Makefile`, we can use it in the CI to ensure there is no drift in the commands being used.

```suggestion
        make setup-tekton-chains
```

---

Oh yeah, I know I was mostly using it as a way to test a few things. I am realizing now that there's no great way to test github action without merging them. So I'm going to test out on local fork first